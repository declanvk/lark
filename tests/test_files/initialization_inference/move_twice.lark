//~ execute:no

struct Bar { x: uint }

def main() {
  let x = Bar(x: 22)
  take(x)
  take(x)
  //~ ERROR: access to uninitialized path
}

def take(v: Bar) { }
