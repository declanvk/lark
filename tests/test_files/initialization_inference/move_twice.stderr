error: access to uninitialized path
- initialization_inference/move_twice:8:7
8 |   take(x)
  |        ^
